spyder/plugins/outlineexplorer/widgets.py

```diff
@@ -597,6 +598,11 @@ def update_tree(self, items, editor):
             symbol_end = symbol_range['end']['line']
             symbol_repr = SymbolStatus(symbol_name, symbol_kind,
                                        (symbol_start, symbol_end), None)
+            if self.symbol_expanded.get(symbol_name):
```

Comment:
```suggestion
            if self._symbols_expanded_state.get(symbol_name):
```

To go with my suggestion above.